src/roles/foreman/tasks/main.yaml

```diff
@@ -173,7 +173,6 @@
       - -c
       - bin/rails db:migrate && bin/rails db:seed
     detach: false
-    working_dir: /usr/share/foreman
```

Comment:
Since we hardcode this path in `Deploy Dynflow Container` play as well, so could we replace it with "$HOME/extras/dynflow-sidekiq.rb"?

Reply:
I can look into this in a separate PR